api/src/main/java/cz/muni/fi/pa165/enums/AuthenticateUserStatus.java

```diff
@@ -0,0 +1,31 @@
+package cz.muni.fi.pa165.enums;
+
+
+
+
+
+public enum AuthenticateUserStatus {
+
+    OK("Mail and password is OK"),
+    UNKNOWN("no find User by email"),
```

Comment:
Unknown status je divne pojmenovani pro neexistujiciho uzivatele. A ta hlaska dava smysl jen na urovni slov, ale korektni veta to neni